### PR TITLE
Replacing Azure storage link with Github links

### LINKED ISSUE #1101: Automate publishing release artifacts to PROD storage account
Now that the manifest file/other release artifacts are being hosted in a PROD storage account (issue #886), we need to automate publishing it there as part of the release.

### PR DESCRIPTION
**Pull request by abhishekkumams**
## Why make this change?

- e.g. Closes #1101 
  - To ease the process for publishing Binaries, manifest and schema file.
  - Since we can github links directly, we won't be publishing to Azure Storage,

## What is this change?

- replacing the Azure storage links with Github release asset download links.

## How was this tested?

- [X] Smoke Test
- [X] Manual

## Sample Request(s)

Every new release in Github is tagged with `latest`. So, the below links always fetch files from the latest release.
**Manifest link:** https://github.com/Azure/data-api-builder/releases/latest/download/dab-manifest.json

**dab-schema link:** https://github.com/Azure/data-api-builder/releases/latest/download/dab.draft.schema.json

### CODE REVIEW

File: schemas/dab.draft.schema.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "title": "Data API Builder",
```
Comment (Aniruddh25) [inline schemas/dab.draft.schema.json:4]:
```suggestion
  "title": "Data API builder",
```

File: schemas/dab.draft.schema.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
   "description": "Schema for Data API Builder engine",
```
Comment (Aniruddh25) [inline schemas/dab.draft.schema.json:5]:
```suggestion
  "description": "Schema for Data API builder engine",
```

**Review by Aniruddh25 — APPROVED** [review]

**Review by Aniruddh25 — APPROVED** [review]
LGTM, thanks for finding this solution and simplifying our release process!!

**Review by ayush3797 — APPROVED** [review]

**Pull request merged**